languages/java/exercises/concept/booleans/.meta/src/refrernce/java/QuestLogic.java

```diff
@@ -0,0 +1,18 @@
+class QuestLogic {
```

Comment:
There is a typo in the folder name `refrernce` instead of `reference`

Reply:
Fixed

Reply:
The folder isn't yet correct, the right name is `reference`

Reply:
sorted